src/analyze/analyze-plot.c

```diff
@@ -196,7 +210,7 @@ static int plot_unit_times(UnitTimes *u, double width, int y) {
         svg_bar("deactivating", u->deactivating, u->deactivated, y);
 
         /* place the text on the left if we have passed the half of the svg width */
-        b = u->activating * SCALE_X < width / 2;
+        b = (u->activating - graph_start_time) * SCALE_X < width / 2;
```

Comment:
Also here.

Reply:
Ditto, `graph_start_time` can be negative.